admin-dev/themes/new-theme/scss/pages/_tax_rules_group_form.scss

```diff
@@ -0,0 +1,3 @@
+.hidden-element {
```

Comment:
Wondering if this file is worth it as it only contains 3 lines of scss. But I dont want to break our conventions. @PierreRambaud wdyt ?

Reply:
We can use bootstrap class `d-none` instead.